docs/data/material/getting-started/templates/checkout/getCheckoutTheme.tsx

```diff
@@ -0,0 +1,651 @@
+              '&:hover': {
+                borderColor: brand[300],
+                boxShadow: `0 0 24px ${brand[100]}`,
+              },
+            }),
+            ...(theme.palette.mode === 'dark' && {
+              backgroundColor: alpha(gray[800], 0.6),
+              outline: `1px solid ${alpha(gray[700], 0.3)}`,
+              ...(ownerState.variant === 'outlined' && {
+                boxSizing: 'border-box',
+                background: `linear-gradient(to bottom, ${gray[900]}, ${alpha(
+                  gray[800],
+                  0.5,
+                )})`,
+                '&:hover': {
+                  borderColor: brand[700],
+                  boxShadow: `0 0 24px ${brand[800]}`,
+                },
+              }),
+            }),
+          }),
+        },
+      },
+      MuiDivider: {
+        styleOverrides: {
+          root: ({ theme }) => ({
+            borderColor: `${alpha(gray[200], 0.8)}`,
+            ...(theme.palette.mode === 'dark' && {
+              borderColor: `${alpha(gray[700], 0.4)}`,
+            }),
+          }),
+        },
+      },
+      MuiPaper: {
+        styleOverrides: {
+          root: ({ theme }) => ({
+            backgroundImage: 'none',
+            backgroundColor: gray[100],
+            ...(theme.palette.mode === 'dark' && {
+              backgroundColor: gray[800],
+            }),
+          }),
+        },
+      },
+      MuiOutlinedInput: {
+        styleOverrides: {
+          notchedOutline: {
+            border: 'none',
+          },
+          root: ({ theme }) => ({
+            '& .MuiInputBase-input': {
+              '&::placeholder': {
+                opacity: 0.7,
+                color: gray[500],
+              },
+            },
+            boxSizing: 'border-box',
+            flexGrow: 1,
+            maxHeight: 40,
+            height: '100%',
+            borderRadius: '10px',
+            border: '1px solid',
+            borderColor: gray[200],
+            boxShadow: '0px 1px 2px rgba(0, 0, 0, 0.1)',
+            transition: 'border-color 120ms ease-in',
+            backgroundColor: alpha(gray[100], 0.4),
+            '&:hover': {
+              borderColor: brand[300],
+            },
+            '&.Mui-focused': {
+              borderColor: brand[400],
+              outline: '4px solid',
+              outlineColor: brand[200],
+            },
+            ...(theme.palette.mode === 'dark' && {
+              '& .MuiInputBase-input': {
+                '&::placeholder': {
+                  opacity: 1,
+                  color: gray[500],
+                },
+              },
+              boxSizing: 'border-box',
+              flexGrow: 1,
+              minHeight: 40,
+              height: '100%',
+              borderRadius: '10px',
+              border: '1px solid',
+              borderColor: gray[700],
+              boxShadow: '0px 2px 4px rgba(0, 0, 0, 1)',
+              backgroundColor: alpha(gray[900], 0.3),
```

Comment:
```suggestion
              backgroundColor: alpha(gray[800], 0.4),
```

I'd bump this one so it's closer to the light mode version and it stands out more from the default background color.